content/references/rippled-api/transaction-formats/transaction-common-fields.md

```diff
@@ -52,8 +52,14 @@ The [`Paths` field](payment.html#paths) of the [Payment transaction][] type can
 
 The `Flags` field can contain various options that affect how a transaction should behave. The options are represented as binary values that can be combined with bitwise-or operations to set multiple flags at once.
 
+To check whether a transaction has a given flag enabled, use a bitwise-and operation comparing the flag's value to the `Flags` field; result of zero indicates the flag is disabled, and a result equal to the flag value indicates the flag is enabled. (Any other result indicates you performed the wrong operation.)
```

Comment:
I know what you're trying to say: `(tfFLAG & flags) == tfFLAG`. How about "...bitwise-and the flag's value and the `Flags` field; result of zero...` The current: "use a bitwise-and operation comparing" transitions into the "comparing" before completing the thought about what we're anding.

Reply:
Modified it slightly because I didn't want to use "bitwise-and" as a verb.